<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>749</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>491</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>PoseEdit</string>
    </property>
    <widget class="QComboBox" name="asset_comboBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>241</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Asset</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="posename_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>241</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Register the pose name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_pushButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>80</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>80</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>491</width>
      <height>511</height>
     </rect>
    </property>
    <property name="title">
     <string>PoseLoad</string>
    </property>
    <widget class="QListWidget" name="pose_listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>471</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="load_comboBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>241</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Load Pose Name</string>
     </property>
    </widget>
    <widget class="QPushButton" name="load_pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>470</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>L O A D</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>491</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>AssetEntry</string>
    </property>
    <widget class="QPushButton" name="entry_pushButton">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>40</y>
       <width>110</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Entry</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="asset_lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>321</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>311</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>fill in the name and select the rig to be registered</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionSetting"/>
   </widget>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSetting">
   <property name="text">
    <string>Setting</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
